diagnostic.md

```diff
@@ -148,10 +171,10 @@ div {
   clear: left;
 }
 ```
-
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+All 'divs' will be positioned at the top of the page, because of float. All
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).